ansible/roles/npm/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+-   name: install node sources
+    shell: curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
```

Comment:
isn't bionic's default node version already 8?

Reply:
I think its better to set the Version because of further updates